### PR TITLE
data: shubhamku044

### PR DESCRIPTION
**Pull request by shubhamku044**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7524"></a>

### CODE REVIEW

File: data/shubhamku044.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/shubhamku044"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://www.youtube.com/@shubhamku044"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (krishstwt) [inline data/shubhamku044.json:26]: Socials do not need `name` property afaik

```suggestion
```

**Review by krishstwt — CHANGES_REQUESTED** [review]
If you want the text, then these need to be links, not social sortcuts, pl. refer to the docs 

- https://linkfree.io/docs/how-to-guides/socials-shortcuts
- https://linkfree.io/docs/how-to-guides/links

File: data/shubhamku044.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "url": "https://www.linkedin.com/in/shubhamku044/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me here for blogging about devrel and other dev topics",
+      "url": "https://dev.to/shubhamku044",
+      "icon": "FaDev"
+    },
```
Comment (RohanSasne) [inline data/shubhamku044.json:34]:
```suggestion
    }
```

Can you please remove the `,`, this is the reason why the json check is failing

**Review by RohanSasne — CHANGES_REQUESTED** [review]

File: data/shubhamku044.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/shubhamku044"
+    },
+    {
+      "icon": "FaYoutube",
+      "url": "https://www.youtube.com/@shubhamku044"
+    },
+    {
```
Comment (CBID2) [inline data/shubhamku044.json:25]: [Recommendation] Consider placing the icons after the URLs. It'll look more appealing.
```suggestion
      "url": "https://twitter.com/shubhamku044"
      "icon": "FaTwitter"
    },
    {
      "url": "https://github.com/shubhamku044"
      "icon": "FaGithub"
    },
    {
      "url": "https://www.youtube.com/@shubhamku044"
       "icon": "FaYoutube"
    },
    {
```

File: data/shubhamku044.json (hunk 2 above)
Comment (CBID2) [inline data/shubhamku044.json:34]: [Recommendation] See my feedback on [this line](https://github.com/EddieHubCommunity/LinkFree/pull/7524/files#r1234535017). They both share the same need for improvement.  
```suggestion
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/shubhamku044/",
      "icon": "FaLinkedin"
    },
    {
      "name": "Follow me here for blogging about devrel and other dev topics",
      "url": "https://dev.to/shubhamku044",
      "icon": "FaDev"
    },
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi, @shubhamku044! :) I just gave you some recommendations on how to improve your Linkfree profile. If you need more suggestions, check out the [profile tips guide](https://linkfree.io/docs/profile-tips) on Linkfree's website. Happy Coding! :)

File: data/shubhamku044.json (hunk 3 above)
Comment (eddiejaoude) [reply data/shubhamku044.json:25]: Thank you @CBID2 for reviewing, it is unclear what you mean by "It'll look more appealing"? People don't look at each other's json file (unless reviewing), they look at the rendered web page - please note this type of feedback on someone's PR is distracting and can be off-putting to the author, so I will mark as resolved

File: data/shubhamku044.json (hunk 2 above)
Comment (eddiejaoude) [reply data/shubhamku044.json]: Please see my comment here https://github.com/EddieHubCommunity/LinkFree/pull/7524/files#r1234671720

File: data/shubhamku044.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      "url": "https://www.linkedin.com/in/shubhamku044/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me here for blogging about devrel and other dev topics",
```
Comment (SaraJaoude) [inline data/shubhamku044.json:30]: As @krishstwt spotted this is not needed:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
This now looks good and I will merge the PR. 

@shubhamku044 you might want to consider in the future adding links to your Profile.  If you do then please raise a new PR - you may also wish to take a look at our [Editing Guide](https://linkfree.io/docs/how-to-guides/editing) in Docs when you do.

**Pull request merged**